### PR TITLE
Raven db 6436

### PR DESCRIPTION
**Pull request by karmeli87**

### CODE REVIEW

File: src/Sparrow/Json/BlittableJsonReaderObject.cs [hunk 1]
```diff
@@ -230,10 +232,19 @@ public bool TryGet<T>(StringSegment name, out T obj)
                 obj = default(T);
                 return false;
             }
-            ConvertType(result, out obj);
+            try
+            {
+                ConvertType(result, out obj);
+            }
+            catch
```
Comment (ayende) [inline src/Sparrow/Json/BlittableJsonReaderObject.cs:239]: Not a good idea, we have a value, just different type, and will mask a lot of errors, I fear

File: src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs [hunk 2]
```diff
@@ -266,5 +266,28 @@ public Task GetReplicationReconnectionQueue()
             }
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/studio-tasks/resolveMerge", "GET", "/databases/{databaseName:string}/studio-tasks/resolveMerge?docId={documentId:string}")]
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs:270]: Need a better name, `suggest-conflict-resolution` ? We aren't resolving anything here.

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Sparrow/Json/BlittableJsonReaderObject.cs (hunk 1 above)
Comment (karmeli87) [reply src/Sparrow/Json/BlittableJsonReaderObject.cs:239]: I think, that if we failed to get a member with the specific type we should return false and not throw an error.

Comment (ayende) [reply src/Sparrow/Json/BlittableJsonReaderObject.cs:239]: No, that will hide issues that will cause trouble later

**Pull request merged**